static/app/actionCreators/projects.tsx

```diff
@@ -358,20 +364,22 @@ export function useAddTeamToProject({
 
       setApiQueryData<Team[]>(
         queryClient,
-        makeProjectTeamsQueryKey({orgSlug, projectSlug}),
-        prevData => (Array.isArray(prevData) ? [...prevData, team] : [team])
+        makeProjectTeamsQueryKey({orgSlug, projectSlug, cursor}),
+        prevData => (Array.isArray(prevData) ? [team, ...prevData] : [team])
```

Comment:
Put the added team at the top of the list since it's easier to see that way.

I wonder if we should invalidate the entire list after doing this?